### PR TITLE
Remove flag and extra 'if' loops.

### PR DESCRIPTION
**Pull request by AlesyaZharnasek**

### CODE REVIEW

File: src/main/java/com/alesya/mafa/pet/MainInterface.java [hunk 1]
```diff
@@ -15,46 +15,37 @@
...
+                    break;
+                case "update":
+                    update();
+                    break;
+                case "delete":
+                    delete();
+                    break;
+                default:
+                    System.out.println(INCORRECT_VALUE);
+                    break;
```
Comment (vitali-ivanou) [inline src/main/java/com/alesya/mafa/pet/MainInterface.java]: do we need it?

File: src/main/java/com/alesya/mafa/pet/MainInterface.java [hunk 2]
```diff
@@ -15,46 +15,37 @@
 
     private IOperationInterface operationInterface = new OperationInterface();
     private Scanner scanner = new Scanner(System.in);
-    private String input;
+    private String input = "";
```
Comment (vitali-ivanou) [inline src/main/java/com/alesya/mafa/pet/MainInterface.java]: Do we need it real field?

File: src/main/java/com/alesya/mafa/pet/MainInterface.java [hunk 3]
```diff
@@ -15,46 +15,37 @@
...
+                    break;
+                case "delete":
+                    delete();
+                    break;
+                default:
+                    System.out.println(INCORRECT_VALUE);
+                    break;
             }
         }
-        scanner.close();
```
Comment (vitali-ivanou) [inline src/main/java/com/alesya/mafa/pet/MainInterface.java:57]: Do we need to close scanner?

Comment (AlesyaZharnasek) [reply src/main/java/com/alesya/mafa/pet/MainInterface.java:57]: done

File: src/main/java/com/alesya/mafa/pet/MainInterface.java (hunk 1 above)
Comment (AlesyaZharnasek) [reply src/main/java/com/alesya/mafa/pet/MainInterface.java]: yes, it is necessary to inform the user that he entered an invalid parameter.

Comment (AlesyaMafa) [reply src/main/java/com/alesya/mafa/pet/MainInterface.java]: done

File: src/main/java/com/alesya/mafa/pet/MainInterface.java (hunk 2 above)
Comment (AlesyaMafa) [reply src/main/java/com/alesya/mafa/pet/MainInterface.java]: done

**Pull request merged**